build.gradle

```diff
@@ -33,6 +34,7 @@ subprojects {
         mavenLocal()
         google()
         jcenter()
+        mavenCentral()
```

Comment:
Why? `jcenter()` is a superset for `mavenCentral()`.

Reply:
Got it.